### PR TITLE
[YUNIKORN-1974] Move to Node.js 18

### PR DESCRIPTION
**Pull request by doupache**
### What is this PR for?
https://nodejs.org/en/blog/announcements/nodejs16-eol
Node 16 has Reached its End-of-Life as of September 11, 2023.

node 18.17 is the latest LTS version.

### What type of PR is it?
* [x] - Improvement

### What is the Jira issue?
[YUNIKORN-1974](https://issues.apache.org/jira/browse/YUNIKORN-1974)

### CODE REVIEW

**Comment by doupache** [comment]
@craigcondit Please approve the CI. I have successfully built the Docker on my local machine with Node 18.17.

File: local-build.sh [hunk 1]
```diff
@@ -36,8 +36,8 @@ function node_version() {
   if [ -r ${NV_FILE} ]; then
     NODE_VERSION=$(<"$NV_FILE")
   fi
-  # docusausrus 2.0.0-beta.18 and later ony work with node 16.14 later, use it as the default
-  NODE_VERSION=${NODE_VERSION:-16.14}
+  # docusausrus 2.0.0-beta.18 and later only work with node 18.17 later, use it as the default
```
Comment (craigcondit) [inline local-build.sh:39]: Rather than an incorrect comment, how about: "Set the node version to use (LTS)"

Comment (craigcondit) [inline local-build.sh:39]: Nit, rather than a factually incorrect comment, just use:

`# Use latest LTS as default`

**Review by craigcondit — CHANGES_REQUESTED** [review]
Minor nit.

**Comment by doupache** [comment]
@craigcondit  I have updated the comment. Please take a look.

**Review by wilfred-s — APPROVED** [review]
LGTM

**Pull request closed without merging**